lib/vm.js

```diff
@@ -54,4 +54,11 @@ exports.runInThisContext = function(code, options) {
   return script.runInThisContext(options);
 };
 
+exports.runInGlobalContext = function(code, options) {
+  var script = new Script(require('module').wrap(code), options);
+  return script.runInThisContext(options)(exports, require, module,
+                                                            __filename,
+                                                            __dirname);
```

Comment:
Sorry, I meant it's not clear to end users what "module" would refer to. But yea, just passing vm's values through isn't great.